src/test/java/htsjdk/samtools/reference/AbstractIndexedFastaSequenceFileTest.java

```diff
@@ -39,13 +36,17 @@
 import java.io.FileInputStream;
 import java.io.FileNotFoundException;
 import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardCopyOption;
 
 /**
  * Test the indexed fasta sequence file reader.
  */
 public class AbstractIndexedFastaSequenceFileTest extends HtsjdkTest {
     private static final File TEST_DATA_DIR = new File("src/test/resources/htsjdk/samtools/reference");
     private static final File SEQUENCE_FILE = new File(TEST_DATA_DIR,"Homo_sapiens_assembly18.trimmed.fasta");
+    private static final File SEQUENCE_FILE_DICT = new File(TEST_DATA_DIR,"Homo_sapiens_assembly18.trimmed.dict");
```

Comment:
New field is unused in this PR, is that intentional? If so, you may want to add a comment as to why it's defined here so it's not deleted later on.

Reply:
good catch, it was used in an earlier version of the test but it turned out to be unecessary